dev/cuda/layernorm_backward.cu

```diff
@@ -20,7 +20,7 @@ version 2 moves a lot of reduction to shared memory over global memory
 
 #define ENABLE_BF16
 #include "common.h"
-
+//typedef unsigned int uint;       // required on windows
```

Comment:
maybe we should just spell the type correctly; uint isn't an actual type in C or C++

Reply:
Good suggestion, let me do it.